Equinox/Globals.h

```diff
@@ -54,5 +56,68 @@ typedef unsigned int uint;
 #define RESIZABLE true
 #define TITLE "Equinox Engine Editor"
 
+inline void DrawBoundingBox(AABB& boundingBox)
```

Comment:
Well it is ok for this concrete case as it is a methot that we use in various places (also note the _inline_ specifier). Take into account that this is not really a class, it is just a header file with some methods defined in it, so they are in the global scope accessible from everywhere. If this file becomes too big, we can hide function implementation in source files, so the header file only keeps the function headers.